src/components/figure/demo/styles-dark.twig

```diff
@@ -0,0 +1,11 @@
+{% embed '../../../themes/demo/fill-canvas.twig' with { theme: 'dark' } %}
+  {% block content %}
+    {% embed '../figure.twig' %}
+      {% if hasCaption %}
+        {% block caption %}
+          This is an image with a caption.
+        {% endblock %}
+      {% endif %}
```

Comment:
It feels like it's probably unnecessary to recreate this content in two different templates?